reana/cli.py

```diff
@@ -1934,10 +1955,21 @@ def python_unit_tests(component):  # noqa: D301
                 " pip install . --upgrade".format(cmd_activate_venv),
                 "git clean -d -ff -x",
                 '{} && pip install ".[tests]" --upgrade'.format(cmd_activate_venv),
-                "{} && python setup.py test".format(cmd_activate_venv),
+                "{} && {} python setup.py test".format(
+                    cmd_activate_venv,
+                    "REANA_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://postgres:mysecretpassword@localhost/postgres"
+                    if does_component_need_db(component)
+                    else "",
+                ),
                 "rm -rf ~/.virtualenvs/_{}".format(component),
             ]:
                 run_command(cmd, component)
+
+            if does_component_need_db(component):
+                run_command(
+                    "echo '==> [INFO] Stopping DB container...' && "
```

Comment:
The `echo` statements should not be needed, the `reana-dev` will print each command that is run via `run_command()` automatically.